cloudinit/config/cloud-init-schema.json

```diff
@@ -1819,11 +1819,6 @@
                   {"type": "array", "items": {"type": "string"}}
                 ]
               }
-            },
-            "squashfuse_in_container": {
-              "type": "boolean",
-              "default": false,
-              "description": "DEPRECATED: This key is no longer needed and will be removed in a future version of cloud-init."
```

Comment:
I guess the moment to drop it is now. Do we have to explicitly say something in the changelog or something else? Or should we keep this for backward compatibility?

Reply:
I think we are clear enough here to drop. It was an optional thing to use in containers until about 2018 with snapd version 2.32.3.1. It is unnecessary and anybody using this config option doesn't really need to anymore. The workaround if someone really wanted to install squashfuse in a container would be to provide `packages: [squashfuse]` in #cloud-config  if necessary for their appllication (which it shouldn't be).